content/docs/intro/languages/javascript.md

```diff
@@ -112,7 +112,7 @@ Tools like VS Code will give you completion lists, live error reporting and inli
 
 When using the built in TypeScript support, Pulumi sets the following compiler settings, which may not be overridden:
 
-- target: "es6",
+- target: "es2016",
```

Comment:
Note: Pulumi actually still defaults to `es6`: https://github.com/pulumi/pulumi/blob/dc2d91e294916a3d74be7d98f1b67e674182b0ee/sdk/nodejs/cmd/run/run.ts#L158

```suggestion
- target: "es6",
```